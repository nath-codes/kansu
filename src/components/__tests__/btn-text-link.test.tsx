import 'jest-styled-components'
import React from 'react'

import ButtonTextLink from '../btn-text-link'
import { renderWithTheme } from './helpers'

describe('ButtonTextLink', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(
      <ButtonTextLink className="foo" href="#" title="bar" />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
