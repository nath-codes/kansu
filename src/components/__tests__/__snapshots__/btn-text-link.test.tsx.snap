// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonTextLink renders correctly 1`] = `
.c0 {
  background: #ffffff;
  border: solid 0.375rem #ffffff;
  color: #202f3c;
  font-family: "Roboto",sans-serif;
  font-size: 1.125rem;
  -webkit-letter-spacing: 2.3px;
  -moz-letter-spacing: 2.3px;
  -ms-letter-spacing: 2.3px;
  letter-spacing: 2.3px;
  line-height: 1;
  padding: 0.5625rem;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-transform: uppercase;
  -webkit-transition: background 0.3s,color 0.3s;
  transition: background 0.3s,color 0.3s;
}

.c0:hover {
  background: none;
  color: #ffffff;
}

<a
  className="foo c0"
  href="#"
  style={
    Object {
      "opacity": 0,
      "transform": "translateY(-1.5rem)",
    }
  }
>
  bar
</a>
`;
